src/sidebar/media-embedder.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 /**
+* Return an HTML5 audio player with the given src URL.
+*/
+
+function audioElement(src) {
+  var html5audio = document.createElement('audio');
+  html5audio.id       = 'audio-player';
+  html5audio.controls = 'controls';
```

Comment:
The `controls` property is a boolean so this should just be `true` rather than `'controls'`. My guess is that this works because non-empty strings when converted to booleans become `true` (and empty strings `false`).